Expensify/App
# Use onSelectRow not to= in ChatLinkRow

cc @Gonals 

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/148663

### Tests
1. Pin a bunch of chats in the LHN
1. Make sure you can switch between them
1. Verify that you are still able to add users to a group chat via the chat switcher
2. Verify you are able to start a chat via the chat switcher by pressing on a chat row

### Screenshots
❌
